src/__tests__/reducers/tasksReducer.test.js

```diff
@@ -0,0 +1,64 @@
+import tasksReducer from '../../redux/reducers/tasksReducer'
+import { SET_SETTINGS, SET_ANSWERS, } from '../../constants'
+
+describe('INITIAL_STATE', () => {
```

Comment:
Imagine, you will have an output: 
```
'INITIAL_STATE ...'
'INITIAL_STATE ...'
'INITIAL_STATE ...'
```
Which doesn't display what actually tested or which test failed